cm-mlops/script/get-generic-python-lib/customize.py

```diff
@@ -38,8 +39,8 @@ def preprocess(i):
                 version = env.get('CM_HOST_OS_VERSION', '')
                 if version:
                     version_split = version.split(".")
-                    if (int(version_split[0]) >= 23) and ('--break-system-packages' not in extra):
-                        extra += '  --break-system-packages '
+                    if (pip_version >= "23.0.0") and ('--break-system-packages' not in extra):
```

Comment:
String comparison using `>=`? This comparison should be done similar to the earlier one used for os version check with 3 parts.